src/context/Data.tsx

```diff
@@ -0,0 +1,13 @@
+import { supabase } from 'services/supabase';
```

Comment:
This is an ambiguous file name.

Reply:
That is on purpose. That way if we change out Supabase or Supabase SWR we don't need to rename this.